Trippy/Views/SelectableMap.m

```diff
@@ -0,0 +1,66 @@
+//
+//  SelectableMap.m
+//  Trippy
+//
+//  Created by Catherine Lu on 7/6/22.
+//
+
+#import "SelectableMap.h"
+#import "Location.h"
+#import "MapUtils.h"
+@import GoogleMaps;
+@import GooglePlaces;
+
+@interface SelectableMap ()
+@property (strong, nonatomic) GMSMapView *mapView;
+@property (strong, nonatomic) NSMutableArray *markersArray;
```

Comment:
nit: you can just call this `markers`. Unless you're planning on having a separate `markersDict` in the future or something.